IdentityCore/src/network/MSIDHttpRequest.m

```diff
@@ -129,4 +156,20 @@ + (void)setRetryIntervalSetting:(NSTimeInterval)retryIntervalSetting { s_retryIn
 + (void)setRequestTimeoutInterval:(NSTimeInterval)requestTimeoutInterval { s_requestTimeoutInterval = requestTimeoutInterval; }
 + (NSTimeInterval)requestTimeoutInterval { return s_requestTimeoutInterval; }
 
+- (NSCachedURLResponse *)cachedResponse
+{
+    NSCachedURLResponse *response = [self.cache cachedResponseForRequest:self.urlRequest];
+    if (response)
```

Comment:
nit: return response would be sufficient here

Reply:
Agree. Addressed.